README.md

```diff
@@ -111,39 +128,63 @@ FAQ
     * Containers run as a non-root unique user that is separate from other system users
       * They cannot access host resources, run privileged, or become root
       * They are given CPU and memory limits defined by the system administrator
-      * Any persistent storage they access will be under a unique SELinux label, which prevents others from seeing their content
-      * These settings are per project, so containers in different projects cannot see each other by default
+      * Any persistent storage they access will be under a unique SELinux label, which
+        prevents others from seeing their content
+      * These settings are per project, so containers in different projects cannot see
+        each other by default
     * Regular users can run Docker, source, and custom builds
-      * By default, Docker builds can (and often do) run as root. You can control who can create Docker builds through the `builds/docker` and `builds/custom` policy resource.
+      * By default, Docker builds can (and often do) run as root. You can control who
+        can create Docker builds through the `builds/docker` and `builds/custom` policy
+        resource.
```

Comment:
Again, the use of fullstops in list items is inconsistent.  (I prefer to use a fullstop always after a full sentence.)